src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -350,34 +418,22 @@ class PeerAPI(override val ipManager: IPManager[IO])(
           complete(StatusCodes.OK)
         }
       }
-    } ~ get {
-      path("snapshot" / "stored") {
-        val storedSnapshots = dao.snapshotStorage.getSnapshotHashes
-        val recentSnapshot = dao.snapshotService.storedSnapshot.get.map(_.snapshot.hash)
-        val hashes = storedSnapshots.flatMap { h =>
-          recentSnapshot.map(hh => List(hh) ++ h)
+    } ~ post {
+      path("snapshot" / "obj" / "snapshot") {
+        entity(as[String]) { ip =>
+          val snapshotHash: IO[Array[Array[Byte]]] =
+            dao.snapshotService.recentSnapshotInfo.lookup(ip.replaceAll("^\"+|\"+$", "")).map { sni =>
```

Comment:
What's the purpose of this `ip.replaceAll`? Additionally, please don't repeat yourself.

Reply:
I had to replace the ip lookups with sending a payload of address to get it working locally, all APIClient's host is localhost. This can all be removed and made as Gets like before, except for "snapshot" / "obj" / "acceptedCBSinceSnapshotCache", which need to be a Post that receives curCheckpointHashes, which is what is currently sent to /snapshot/info